x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -54,9 +54,10 @@ export const AGENT_POLICY_SUMMARY_ROUTE = `${BASE_POLICY_ROUTE}/summaries`;
 export const ISOLATE_HOST_ROUTE = `${BASE_ENDPOINT_ROUTE}/isolate`;
 export const UNISOLATE_HOST_ROUTE = `${BASE_ENDPOINT_ROUTE}/unisolate`;
 
-/** Endpoint Actions Log Routes */
+/** Endpoint Actions Routes */
 export const ENDPOINT_ACTION_LOG_ROUTE = `/api/endpoint/action_log/{agent_id}`;
 export const ACTION_STATUS_ROUTE = `/api/endpoint/action_status`;
+export const ACTION_DETAILS_ROUTE = `/api/endpoint/action/{action_id}`;
```

Comment:
GET using Paul's original route here is more REST-y. I'd leave it as is

Reply:
yeah, I was/am going for a more REST like API path.

When I looked at the routes I noticed that some were being created in a "flat" way instead of a more "REST" way.  IMO, stuff associated with "actions"'s could have followed a structure such as: `/api/endpoint/action/{purpose}/**`. Example:

```
// Create an action
POST /api/endpoint/action

// List of actions
GET /api/endpoint/action/find

// Get Details of an action
GET /api/endpoint/action/item/{action_id}

// Get action status for a set of endpoints
GET /api/endpoint/action/status

// Get action log list for an endpoint agent id
GET /api/endpoint/action/log/{agent_id}
```

Reply:
We should update the errant paths at some point then 👍